src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx

```diff
@@ -32,30 +33,27 @@ class OrganizationProjectsView extends OrganizationSettingsView {
 
         <table className="table table-no-top-border m-b-0">
           <tbody>
-            {projects
-              .sort(({isBookmarked}) => (isBookmarked ? -1 : 1))
-              .map(project => (
-                <ProjectListItem
-                  key={project.slug}
-                  project={project}
-                  organization={this.context.organization}
-                />
-              ))}
+            {sortProjects(projects).map((project, i) => (
+              <ProjectListItem
+                key={i}
```

Comment:
Better to avoid using array indices as a key

Reply:
because it doesn't catch logical dupes?

Reply:
Because it doesn't uniquely identify the elements in the array, so when things get added/removed from the array, react assumes things with the same index have not changed: https://reactjs.org/docs/lists-and-keys.html